components/CreateNewCampaign/MultipleFileUploadField.tsx

```diff
@@ -0,0 +1,121 @@
+import { Grid, Typography } from '@mui/material';
+import { useField } from 'formik';
+import React, { useCallback, useEffect, useState } from 'react';
+import { FileError, FileRejection, useDropzone } from 'react-dropzone';
+import SingleFileUploadWithProgress from './SingleFileUploadWithProgress';
+import { UploadError } from './UploadError';
+
+// import { makeStyles } from '@mui/styles';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+
+let currentId = 0;
+
+function getNewId() {
+  return ++currentId;
+}
+
+export interface UploadableFile {
+  id: number;
+  file: File;
+  errors: FileError[];
+  url?: string;
+}
+
+const theme = createTheme();
+
+// const useStyles = makeStyles((theme) => ({
+//   dropzone: {
+//     border: `2px dashed ${theme.palette.primary.main}`,
+//     borderRadius: theme.shape.borderRadius,
+//     display: 'flex',
+//     alignItems: 'center',
+//     justifyContent: 'center',
+//     background: theme.palette.background.default,
+//     height: theme.spacing(10),
+//     outline: 'none',
+//   },
+// }));
+
+const styledTypo = {
+  textAlign: 'center',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+}
+
+
+const MultipleFileUploadField = ({ name }: { name: string }) => {
+  const [_, __, helpers] = useField(name);
+//   const classes = useStyles();
+
+  const [files, setFiles] = useState<UploadableFile[]>([]);
+  const onDrop = useCallback((accFiles: File[], rejFiles: FileRejection[]) => {
+    const mappedAcc = accFiles.map((file) => ({ file, errors: [], id: getNewId() }));
+    const mappedRej = rejFiles.map((r) => ({ ...r, id: getNewId() }));
+    setFiles((curr) => [...curr, ...mappedAcc, ...mappedRej]);
```

Comment:
think we can do away with id and just use the file's name as the key for ur map below, and we should not allow file with same name to be uploaded. 
https://stackoverflow.com/questions/26296232/dropzone-prevent-uploading-of-duplicate-files